service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -105,30 +107,35 @@ void aboutToStartShouldReturnErrorsWhenFamilymanNumberIsNotProvided() throws Exc
     void midEventShouldGenerateOrderDocument() throws Exception {
         byte[] pdf = {1, 2, 3, 4, 5};
         Document document = document();
-        DocmosisDocument docmosisDocument = new DocmosisDocument(C21.getDocumentTitle(), pdf);
+        DocmosisDocument docmosisDocument = new DocmosisDocument("order.pdf", pdf);
 
         given(docmosisDocumentGeneratorService.generateDocmosisDocument(any(), any())).willReturn(docmosisDocument);
-        given(uploadDocumentService.uploadPDF(USER_ID, AUTH_TOKEN, pdf, C21.getDocumentTitle()))
+        given(uploadDocumentService.uploadPDF(eq(USER_ID), eq(AUTH_TOKEN), eq(pdf), any()))
```

Comment:
are we not able to put a document title in here instead of 'any' ?

Reply:
the document title will be either 'care_order.pdf' or 'blank_order_c21.pdf', I could do a given for both scenarios though 👍 that's better than any()

Reply:
cool maybe go with two givens then :)

Reply:
I'll go with one given for now ("blank_order_c21.pdf") as I don't think we need a mid event test for every type of document (sounds like too much testing the same thing). If it can generate one, it can generate all of them, but good to assert that the filename was correct instead of any()